routes/shopItem.js

```diff
@@ -1,22 +1,24 @@
 const express = require("express");
 const utility = require("../utility.js");
+const db = require("../server.js");
 
 shopItemRouter = express.Router();
 
-shopItemRouter.use(utility.readDatabaseMiddleware);
-
 shopItemRouter.post("/", (req, res, next) => {
     try {
         if ("isCompleted" in req.body === false) {
             req.body.isCompleted = false;
         }
         if ((typeof req.body.name) === "string" && (typeof req.body.isCompleted) === "boolean") {
-            req.body.id = req.database.nextId;
-            req.database.nextId += 1;
-            req.database.shopItems.push(req.body);
-            utility.writeDatabase(req.database);
-            res.json(`${req.method} success`);
-            res.status(201).send();
+            db.run("INSERT INTO grocery_list (name) VALUES(?)",
```

Comment:
You should add the value of isCompleted  as well as we did with the previous (JSON) database